src/components/Comments.jsx

```diff
@@ -1,5 +1,32 @@
-const Comments = () => {
+import axios from "axios"
+import {useEffect, useState} from "react"
+import {useParams} from "react-router-dom"
 
+const Comments = () => {
+    const [comments, setComments] = useState([])
+    const {article_id} = useParams()
+    useEffect(() => {
+        axios.get(`https://news-williammason.herokuapp.com/api/articles/${article_id}/comments`).then((data) => {
+            console.log(data.data.comments)
+            setComments(data.data.comments)
+        }).catch((err) => {
+            console.log(err.response)
+        })
+    }, [article_id])
+    return <section>
+        <h3>Comments</h3>
+        <ul>
+            {comments.map((comment) => {
+                return <li key={comment.comment_id}>
+                    <h4>{comment.author}</h4>
+                    {comment.body}
+                    <br></br>
+                    Votes: {comment.votes}
```

Comment:
Currently we're just inserting the comment.body and the votes directly into the html but they should be wrapped in tags such as \<p>{comment.body}\<p>

I'd also maybe look at getting even just some simple styling built into the project as this will help us with the layout instead of using \<br>\</br> as well as being able to get rid of the bullet points next to each comment